README.md

```diff
@@ -33,6 +46,12 @@ For more detailed information about PostHTML in general take a look at the [docs
       <br />
       <a href="https://github.com/voischev">Ivan Voischev</a>
     </td>
+     <td align="center">
+      <img width="150 height="150"
+      src="https://github.com/anikethsaha.png?size=150">
+      <br />
+      <a href="https://github.com/anikethsaha">Anix Anikethsaha</a>
```

Comment:
```suggestion
      <a href="https://github.com/anikethsaha">Aniketh Saha</a>
```

Its a bit confusing have different username and name  😄

Reply:
Sorry

Reply:
no issue, I should have a single username 👍 . its confusing for others as well.